common-crafting.lic

```diff
@@ -346,5 +346,30 @@ module DRCC
       end
     end
   end
-    
+  
+  def check_for_existing_sigil?(sigil, stock_number, quantity, bag, belt, info)
+    merged = []
+    primary = []
+    secondary = []
+    primary = ["abolition", "congruence", "induction", "permutation", "rarefaction"]
+    secondary = ["antipode", "ascension", "clarification", "decay", "evolution", "integration", "metamorphosis", "nurture", "paradox", "unity"]
+    merged.concat primary
+    merged.concat secondary
```

Comment:
Can this entire section be consolidated to something like:
```ruby
primary = ["abolition", "congruence", "induction", "permutation", "rarefaction"]
secondary = ["antipode", "ascension", "clarification", "decay", "evolution", "integration", "metamorphosis", "nurture", "paradox", "unity"]
merged = primary + secondary
```
I assume you're keeping the two primary and secondary arrays just so it's easy to see which sigils are primary vs. secondary when reading the code, otherwise I think this section could be consolidated to a single line.

Reply:
I tried that and it did not work. I can try again later. The only way I could get them together and flattened was how I did it. Should I add the Guild sigils now even though they are not released?

Reply:
Try
```
merged = [primary + secondary].flatten
```
That declares the array at the same time you populate it. 

Larger question: if all of these are now purchaseable, whats the sense in having two lists?

Reply:
There is no need to flatten two 1-D arrays that are simply concatenated, which is what the `+` operator does.

![image](https://user-images.githubusercontent.com/6467969/186504041-a0d87c4f-e0d6-4269-abf7-cc962bc59886.png)


If we don't need the two arrays for some code readability/understanding sake, this whole section turns into a one-line array assignment.

Reply:
Today `merged = primary + secondary` worked. Last night it was giving me them combined but like this: `[["abolition", "congruence", "induction", "permutation", "rarefaction"], ["antipode", "ascension", "clarification", "decay", "evolution", "integration", "metamorphosis", "nurture", "paradox", "unity"]]` :man_shrugging: 

Anyway, should I add in the Guild ones now?